ejson/src/payload-converter.ts

```diff
@@ -0,0 +1,5 @@
+// @@@SNIPSTART typescript-ejson-converter
+import { EjsonPayloadConverter } from './ejson-payload-converter';
+
+export const payloadConverter = new EjsonPayloadConverter();
```

Comment:
We might want to show how to use the composite converter here instead of just a single one.

Reply:
Ah, it doesn't support `undefined`. Will add